pre_commit_hooks/update_commit_msg.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, unicode_literals
+import sys
+import optparse
+from invoke import run
+
+import re
+
+
+class UpdateCommitMsg(object):
+    """
+    This hook saves developers time by prepending ticket numbers to commit-msgs.
```

Comment:
v v minor, docstrings shouldn't start with whitespace, (and single-line shouldn't end with whitespace)

```python
"""yes"""

"""yes

yes
"""

"""no
"""

"""
no
no
"""
```